### PR TITLE
Feature/nex 132 vendorization of qti item runner

### PR DESCRIPTION
**Pull request by btamas**
Vendorization of tao qti item runner.

Jira: https://oat-sa.atlassian.net/browse/NEX-132

Post task:
- [ ] release to npm

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    "bugs": {
+        "url": "https://github.com/oat-sa/tao-item-runner-qti-fe/issues"
+    },
+    "homepage": "https://github.com/oat-sa/tao-item-runner-qti-fe#readme",
+    "devDependencies": {
+        "@oat-sa/browserslist-config-tao": "^0.1.0",
+        "@oat-sa/tao-core-libs": "^0.1.0",
+        "@oat-sa/tao-core-sdk": "^0.3.0",
+        "@oat-sa/tao-core-ui": "^0.1.1",
+        "@oat-sa/tao-item-runner": "github:oat-sa/tao-item-runner-fe#feature/NEX-132_vendorization-of-item-runner",
```
Comment (ferenckiss89) [inline package.json]: this branch does not exist anymore

File: build/rollup.config.js [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+            'lib/gamp/gamp',
+            'class',
+            'mathJax',
+            'nouislider',
+            'interact',
+            'select2',
+            'ckeditor',
+            'iframeNotifier',
+
+            'taoQtiItem/qtiCreator/model/variables/OutcomeDeclaration',
```
Comment (krampstudio) [inline build/rollup.config.js]: Which module requires something from the authoring ?

File: build/rollup.config.js [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+            'taoQtiItem/qtiCreator/helper/qtiElements',
```
Comment (krampstudio) [inline build/rollup.config.js]: Same here, could you please let me know which module requires this helper ?

File: portableLib/OAT/customEvent.js [hunk 4]
```diff
@@ -0,0 +1,110 @@
+/**
```
Comment (krampstudio) [inline portableLib/OAT/customEvent.js]: Why do you need to have a copy of the `portableLib`.
FYI  `portableLib` are libraries proposed to PCI and PIC implementer to be reused in their custom interaction/info control. 
If this is only for the tests, we will need to find a way to not have them part of this repository.

File: build/rollup.config.js (hunk 2 above)
Comment (btamas) [reply build/rollup.config.js]: It is used in `qtiItem/helper/maxScore.js`: https://github.com/oat-sa/tao-item-runner-qti-fe/blob/feature%2FNEX-132_vendorization-of-item-runner/src/qtiItem/helper/maxScore.js#L114

File: build/rollup.config.js (hunk 3 above)
Comment (btamas) [reply build/rollup.config.js]: It was a mistake. I removed it.

File: portableLib/OAT/customEvent.js (hunk 4 above)
Comment (btamas) [reply portableLib/OAT/customEvent.js]: It was necessary for `portableElementRegistry`, but it is not in this repository anymore. So I removed them.

File: build/rollup.config.js (hunk 2 above)
Comment (krampstudio) [reply build/rollup.config.js]: Ok, let me investigate. We will have to refactor it otherwise we will have an issue

Comment (krampstudio) [reply build/rollup.config.js]: I think you can use `taoQtiItem/qtiItem/core/variables/OutcomeDeclaration` instead (please verify the API) but the developer use the overridden version from the authoring by mistake

**Pull request merged**